editoast/src/core/pathfinding.rs

```diff
@@ -9,15 +9,18 @@ use utoipa::ToSchema;
 
 use crate::core::{AsCoreRequest, Json};
 use crate::error::InternalError;
+use crate::views::path::pathfinding::PathfindingFailure;
+use crate::views::path::pathfinding::PathfindingResult;
```

Comment:
We want `views` to depend on `core` for the split in crates, can we move the code that uses these definitions in `views` instead?